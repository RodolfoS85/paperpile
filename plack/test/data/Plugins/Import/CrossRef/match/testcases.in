---
test_comment: "INCOLLECTION"
doi: 10.1093/0199288917.003.0002
---
test_comment: "INPROCEEDINGS"
doi: 10.1109/ICPR.2006.501
---
test_comment: "INCOLLECTION"
doi: 10.1007/978-3-642-14538-4
---
test_comment: "PROCEEDINGS"
doi: 10.1007/978-3-642-00328-8
---
test_comment: "ARTICLE"
doi: 10.1126/science.1112009
---
test_comment: "ARTICLE"
doi: 10.1101/gr.4137606
---
test_comment: "BOOK"
doi: 10.1037/10828-007


