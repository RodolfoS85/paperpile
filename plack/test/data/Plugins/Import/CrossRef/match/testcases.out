---
address: Oxford, England
authors: Swinnen, Johan F. M.
chapter: 2. Patterns of Transition
doi: 10.1093/0199288917.003.0002
editors: Swinnen, Johan F. M. and Rozelle, Scott
isbn: '0199288917'
month: '01'
pages: 13-28
publisher: Oxford University Press
pubtype: INCOLLECTION
title: From Marx and Mao to the Market
volume: '1'
year: '2006'
---
authors: Stork, D. G. and Johnson, M. K.
booktitle: 18th International Conference on Pattern Recognition (ICPR'06)
doi: 10.1109/ICPR.2006.501
isbn: 0-7695-2521-0
pages: 255-258
publisher: IEEE
pubtype: INPROCEEDINGS
title: Estimating the location of illuminants in realist master paintings Computer image analysis addresses a debate in art history of the Baroque
year: '2006'
---
address: Berlin, Heidelberg
authors: Montali, Marco
doi: 10.1007/978-3-642-14538-4
editors: Aalst, Will and Mylopoulos, John and Sadeh, Norman M. and Shaw, Michael J. and Szyperski, Clemens
isbn: 978-3-642-14537-7
issn: 1865-1348
publisher: Springer Berlin Heidelberg
pubtype: INCOLLECTION
series: Lecture Notes in Business Information Processing
title: Specification and Verification of Declarative Open Interaction Models
volume: '56'
year: '2010'
---
address: Berlin, Heidelberg
authors: Ardagna, Danilo and Mecella, Massimo and Yang, Jian
doi: 10.1007/978-3-642-00328-8
editors: Aalst, Will and Mylopoulos, John and Sadeh, Norman M. and Shaw, Michael J. and Szyperski, Clemens
isbn: 978-3-642-00327-1
issn: 1865-1348
publisher: Springer Berlin Heidelberg
pubtype: PROCEEDINGS
series: Lecture Notes in Business Information Processing
title: Business Process Management Workshops
volume: '17'
year: '2009'
---
authors: '{RIKEN Genome Exploration Research Group and Genome Science Group (Genome Network Project Core Group) and the FANTOM Consortium}'
doi: 10.1126/science.1112009
issn: 0036-8075
issue: '5740'
journal: Science
month: '09'
pages: 1564-1566
pubtype: ARTICLE
title: Antisense Transcription in the Mammalian Transcriptome
volume: '309'
year: '2005'
---
authors: Akiva, P.
doi: 10.1101/gr.4137606
issn: 1088-9051
issue: '1'
journal: Genome Research
month: '12'
pages: 30-36
pubtype: ARTICLE
title: Transcription-mediated gene fusion in the human genome
volume: '16'
year: '2005'
---
address: Washington, DC, US
authors: Patenaude, Andrea Farkas
doi: 10.1037/10828-007
isbn: 1-59147-110-9
publisher: American Psychological Association
pubtype: BOOK
title: A Family Matter
year: '2005'
